Source/WebKit/UIProcess/_WKTouchEventGenerator.mm

```diff
@@ -48,7 +47,7 @@
 
 NSUInteger const HIDMaxTouchCount = 5;
 
-static int fingerIdentifiers[HIDMaxTouchCount] = { 2, 3, 4, 5, 1 };
+static constexpr std::array<int, 5> fingerIdentifiers { 2, 3, 4, 5, 1 };
```

Comment:
Thanks for switching to `constexpr`.

Does not seem like an improvement to move from `HIDMaxTouchCount` to `5` here. If we are specifying a size, I think it’s great to document that must match `HIDMaxTouchCount`. Or if we instead want to omit the size then, since the values are all `int` constants, this can just be `std::array`, with no template arguments.

@aproskuryakov has also pointed out to me a few times that in cases like this we only need `constexpr`, don’t also need `static`.

Reply:
Oh, that's a mistake, I intended to use `HIDMaxTouchCount` instead of 5 here, my bad.